pkg/controllers/management/eks/eks_cluster_handler.go

```diff
@@ -44,6 +45,7 @@ import (
 	"k8s.io/client-go/kubernetes"
 	"k8s.io/client-go/rest"
 	"sigs.k8s.io/aws-iam-authenticator/pkg/token"
+	yaml2 "sigs.k8s.io/yaml"
```

Comment:
Cosmetic comment/question: the [latest YAML release](https://pkg.go.dev/sigs.k8s.io/yaml) is ```1.2.0```. Is there a reason why we use ```yaml2``` here?

Reply:
I think GoLand did this to avoid colliding with the variable at L675. Will change, good catch.